### DISCUSSION TITLE
Date & Time, Anno Domini Year

### DISCUSSION THREAD

**Discussion by ghost** [MarcinOrlowski/html-clock-plasmoid#62] [HTML Clock templates]
Minimalist right aligned 12-hour time and date with Anno Domini labelled year.

![clock](https://user-images.githubusercontent.com/8041294/220184771-95ba735e-14de-40fc-812d-f717ce2a272d.png)

```html
<html>
<body>
<table style="border: none;" align="right">
<tr>
  <td>
    <table style="border: none;">
      <tr><td align="right"><span style="font-weight: bold; color: white;">{k}:{ii} {AA}</span></td></tr>
      <tr><td align="right"><span style="font-weight: bold; color: white;">{DD}, {MM} {d}</span></td></tr>
      <tr><td align="right"><span style="font-weight: bold; color: white;">Anno Domini {yyyy}</span></td></tr>
    </table>
  </td>
</tr>
</table>
</body>
</html>
```

**Confirmation**
* [x] I confirm this is my original work.
* [x] I attached screenshot of my layout to this ticket.
* [x] I grant full and unrevocable rights to use the layout with HTML Clock widget free of charge.
* [x] I grant users of HTML Clock widget right to use my layout free of charge and freely alter my layout if they need to.
* [x] I want to be credited as [Dravenus Rex.](https://gab.com/DravenusRex)
* [x] This layout is licensed under the MIT License.

**Comment by MarcinOrlowski** [comment]
Thanks for your contribution. I edited your template and removed hardcoded font size to allow copy-pasters to be able to control that from plasmoid's settings. Also added a screenshot of your layout.
